Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,44 @@
 ﻿using System;
+using System.Diagnostics;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5141367)